cypress/e2e/3-learn/home.cy.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+describe('my awesome website', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+
+  it('turns page to dark mode', () => {
+    cy.get('.Theme-Toggle').check()
```

Comment:
Follow the check of the toggle a with a second chained function.
Would be along the lines of:
`cy.get('.The-Class-That-Has-The-Background').should("have.css", "background-color", "rgb(the-colour-code-to-check-for")`